### ISSUE TITLE
mixin: failed injection: Lnet/minecraft/world/World

### ISSUE THREAD

**Issue by DMahalko** [SpongePowered/SpongeForge#2255]
- Summary:
Still trying to get modpack "Hexxit Updated" working. Disabled malisiscore and associated mods, as they are just decorative eye candy and don't affect core gameplay. Renamed spongeforge from "aaa-" back to normal, no effect, still crashes.

- Everything appears to be working fine right up to line 2242:
[11:12:35] [main/WARN] [mixin]: Failed to locate initialiser injection point in <init>(Lnet/minecraft/world/World;)V, initialiser was not mixed in.

- SpongeForge version: spongeforge-1.12.2-2705-8.0.0-BETA-3225.jar
- Forge version: Forge Mod Loader version 14.23.4.2705 for Minecraft 1.12.2
- Java version:  Java HotSpot(TM) 64-Bit Server VM, version 1.8.0_144
- Operating System: Linux:amd64:4.4.0-91-generic

- FTB/Curse/Twitch modpack: Hexxit Updated
Modpack download: https://minecraft.curseforge.com/projects/hexxit-updated/files/2568136

- Plugins/Mods:
1.12.2\TheBetweenlands-3.3.8-core.jar
AbyssalCraft-1.12.2-1.9.4.9.jar
aether_legacy-1.12.2-v3.0.jar
AIImprovements-1.12-0.0.1b3.jar
AutoRegLib-1.3-17.jar
backpack-3.0.2-1.12.2.jar
BattleTowers-1.12.2.jar
BiblioCraft[v2.4.4][MC1.12.2].jar
BiomesOPlenty-1.12.2-7.0.1.2391-universal.jar
CarryOn+MC1.12.2+v1.9.jar
Chameleon-1.12-4.1.3.jar
Chisel-MC1.12.2-0.2.0.31.jar
chiselsandbits-14.17.jar
chococraft_1.12.1-0.9.1.52.jar
ClassicCombat-1.12-1.0.0.jar
conarm-1.12.2-0.0.19-b.jar
CraftIRC.jar
dailies-1.12.2-6.jar
DimensionalDoors-3.0.9-287.jar
DungeonTactics-1.12.2-0.14.2.jar
ElevatorMod-1.12.2-1.3.6.jar
Erebus-1.0.22.jar
foamfix-0.9.9.1-1.12.2-anarchy.jar
FoodExpansion1.3.2-1.12.jar
FTBLib-5.1.0.19.jar
FTBUtilities-5.1.0.24.jar
furniture-4.2.0-1.12.2.jar
gravestone-1.10.1.jar
griefprevention-1.12.2-4.3.0.555.jar
Hexxit+Gear-1.12.2-2.7.0.jar
InfernalMobs-1.12.2.jar
ironchest-1.12.2-7.0.42.827.jar
IvToolkit-1.3.3-1.12.jar
jei_1.12.2-4.9.2.196.jar
journeymap-1.12.2-5.5.2.jar
JRFTL[1.12.2]-1.1.jar
LuckPerms-Sponge-4.1.9.jar
MagmaMonsters-0.3.0.jar
Mantle-1.12-1.3.2.24.jar
mtqfix-1.12.2-1.1.0.jar
natura-1.12.2-4.3.2.49.jar
Nucleus-1.3.1-S7.0-MC1.12.2-plugin.jar
ObsidianPlates-1.3.0_1.12.jar
Ore+Tweaker-1.0.2-1.12.0.jar
Placebo-1.12.2-1.3.4.jar
Quark-r1.4-123.jar
RecurrentComplex-1.4.7.jar
Reliquary-1.12.2-1.3.4.763.jar
RoguelikeDungeons-1.12.2-1.8.0.jar
secretroomsmod-1.12.2-5.6.4.jar
SoulShardsRespawn-1.12.2-1.0.5-7.jar
SpartanShields-1.12.2-1.4.1.jar
spongeforge-1.12.2-2705-8.0.0-BETA-3225.jar
StorageDrawers-1.12.2-5.3.6.jar
TConstruct-1.12.2-2.10.1.84.jar
TheBetweenlands-3.3.8-universal.jar
theoneprobe-1.12-1.4.22.jar
TickDynamic-1.12.2-1.0.1.jar
topaddons-1.12.2-1.7.0.jar
toroquest-1.12.2-5.2.jar
TreeChopper-1.12.2-1.2.4.jar
twilightforest-1.12.2-3.7.424-universal.jar
Waddles-1.12.2-0.6.0.jar

- **Disabled** mods, just trying to get the rest of the modpack to launch
malisiscore-1.12.2-6.4.0.jar.disabled
malisisdoors-1.12.2-7.3.0.jar.disabled
malisisswitches-1.12.2-5.1.0.jar.disabled

debug.log:
https://gist.github.com/DMahalko/948ba099e54c05e4dd74c409fb06ee23

**Comment by gabizou** [comment]
Can you try using the latest 7.1.0 SpongeForge build 3228? You're using a SpongeForge built for SpongeAPI 8 which has lots of breaking changes and is targeting 1.13 at some point, so all recommended builds are using API 7.x releases, which in this case the latest SpongeForge build is 3228.

**Comment by DMahalko** [comment]
Followup to anyone else doing a Google search for this error. After trying every sponge I can find, I decided to start disabling mods one at a time until this startup crash went away, and I found the problem... 

**TickDynamic-1.12.2-1.0.1.jar**

https://minecraft.curseforge.com/projects/tick-dynamic

_Tick Dynamic is a Minecraft Forge CoreMod, which will attempt to maintain a server Ticks Per Second at 20. It does this by individually controlling how many Entities and TileEntities update each tick. As the server TPS goes down, the number of Entities and TileEntities that update each tick also goes down, to maintain a high server TPS._

_**Note: this is not currently SpongeForge compatible. I am working to resolve the issue**._
